front/app/modules/commercial/widgets/admin/containers/index.tsx

```diff
@@ -1,22 +1,40 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import IdeasWidget from './IdeasWidget';
 import { SectionTitle, SectionDescription } from 'components/admin/Section';
+import { Box, colors } from '@citizenlab/cl2-component-library';
+import { useLocation } from 'react-router-dom';
+import clHistory from 'utils/cl-router/history';
+import GoBackButton from 'components/UI/GoBackButton';
 
 import messages from '../messages';
 import { FormattedMessage } from 'utils/cl-intl';
 
-export default class AdminSettingsWidgets extends PureComponent {
-  render() {
-    return (
-      <>
-        <SectionTitle>
-          <FormattedMessage {...messages.titleWidgets} />
-        </SectionTitle>
-        <SectionDescription>
-          <FormattedMessage {...messages.subtitleWidgets} />
-        </SectionDescription>
+const AdminSettingsWidgets = () => {
+  const location = useLocation();
+
+  const goBack = () => {
+    clHistory.goBack();
+  };
+  const hasGoBackLink = location.key !== 'default';
```

Comment:
Let's move vars about methods? :-)

Reply:
Also, this seems something we need to streamline across the app. Can you check with Iva?

Reply:
Not sure I follow, what do you mean "move vars about methods"

This is merely determining whether to show the back button or not. Are you suggesting building a back button with that functionality?
> Also, this seems something we need to streamline across the app

Reply:
> Not sure I follow, what do you mean "move vars about methods"

Generally speaking, we put vars above method in functional components. No strict rule, but easier to quickly grasp components if most follow ~ same pattern. 

> This is merely determining whether to show the back button or not. Are you suggesting building a back button with that functionality?

Was only wondering if this should happen manually in each file, vs. a more generalized system. Feels wrong in some way, but I don't have a concrete proposition + may need a bit more instances of manually checking hasGoBackLink before abstracting.